apps/api/src/app/auth/usecases/register/user-register.usecase.ts

```diff
@@ -37,7 +36,6 @@ export class UserRegister {
     });
 
     let organization: OrganizationEntity;
-    let member: MemberEntity;
     if (command.organizationName) {
```

Comment:
@davidsoderberg So actually, we are not generating the organization in the user-register flow, since he might create an account using GitHub. I think it will be better to tie the environment created to be inside the OrganizationCreate usecase to cover all cases of registration.  Wdyt?